### ISSUE TITLE
How to specify data type in html using attribute html 

### ISSUE THREAD

**Issue by PouriaSeyfi** [PHPOffice/PhpSpreadsheet#3444]
During HTML to Excel conversion, it is sometimes necessary to know what date type should be used :  

```html
                <table>
                    <tr>
                        <!-- boolean data type -->
                        <td data-type="b">1</td>
                        <!-- string data type -->
                        <td data-type="s">12345678987654321</td>
                        <!-- formula data type -->
                        <td data-type="f">=CONCAT("TEXT A ","TEXT B")</td>
                    </tr>
                </table>
````

**Comment by PouriaSeyfi** [comment]
Please review this PR 
@MarkBaker

**Comment by oleibman** [comment]
Resolved by PR #3445.
